salt/config/__init__.py

```diff
@@ -1024,6 +1029,8 @@
     # ZMQ HWM for EventPublisher pub socket - different for minion vs. master
     'event_publisher_pub_hwm': 1000,
     'event_match_type': 'startswith',
+    # If a proxymodule has a function called 'grains' then call it and merge
+    # the results with the rest of the grains
```

Comment:
Is this supposed to have a default value for `proxy_merge_grains_in_module` here?